### PR TITLE
[issue #68]add new module to support swagger2

### PR DESCRIPTION
**Pull request by GuoYL123**

### CODE REVIEW

**Comment by GuoYL123** [comment]
DO NOT MERGE

File: spring-cloud-huawei-sample/discovery-demo/price-provider/pom.xml [hunk 1]
```diff
@@ -21,12 +21,11 @@
   <parent>
     <groupId>com.huaweicloud</groupId>
     <artifactId>discovery-demo</artifactId>
-    <version>0.0.3</version>
+    <version>0.0.4</version>
```
Comment (liubao68) [inline spring-cloud-huawei-sample/discovery-demo/price-provider/pom.xml]: 修改版本号不要和这个需求一起合入。 发布版本后续需要保持比较正规的版本计划和规范，不随便发布版本。

Comment (GuoYL123) [reply spring-cloud-huawei-sample/discovery-demo/price-provider/pom.xml]: done

**Comment by GuoYL123** [comment]
do not merge

File: spring-cloud-huawei-sample/swagger-demo/springcloud-provider/pom.xml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+      <version>${spring-cloud.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>com.huaweicloud</groupId>
+      <artifactId>spring-cloud-starter-huawei-servicecomb-discovery</artifactId>
+      <version>${project.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>com.huaweicloud</groupId>
+      <artifactId>spring-cloud-starter-huawei-swagger2</artifactId>
```
Comment (liubao68) [inline spring-cloud-huawei-sample/swagger-demo/springcloud-provider/pom.xml]: How about spring-cloud-starter-huawei-swagger? I think it's not necessary to mention the version.

Comment (GuoYL123) [reply spring-cloud-huawei-sample/swagger-demo/springcloud-provider/pom.xml]: done

**Comment by GuoYL123** [comment]
Swagger generation is done.
Exchange with Java-chassis will be done in the next PR.

**Pull request merged**